packages/flare/bin/flare.py

```diff
@@ -99,16 +96,11 @@ def _retrieve_event_feed_metadata(
             # Rate limiting.
             time.sleep(1)
 
-    def _enrich_event_from_metadata_uid(self, *, event: dict) -> dict:
-        if event["metadata"] and event["metadata"]["uid"]:
-            event_response = self.flare_client.get(
-                url=f"/firework/v2/activities/{event['metadata']['uid']}"
-            )
-            enriched_event = event_response.json()
-            self.logger.debug(enriched_event)
-            return enriched_event
-        else:
-            return event
+    def _retrieve_full_event_from_uid(self, *, uid: str) -> dict:
+        event_response = self.flare_client.get(url=f"/firework/v2/activities/{uid}")
+        event = event_response.json()
```

Comment:
Are you sure we shouldn't return event_response["event"] here?

Reply:
Do you mean event_response["activity"]?

Reply:
Yes